### DISCUSSION TITLE
python3 버전

### DISCUSSION THREAD

**Discussion by cueon** [DKU-SWExpert-K8s/K8s#4] [함께 나누기]
ubuntu 20.04 에서 python3.8 을 제공하는데

책에서 clone 받으라고 하는 [kubespray 레포지토리](https://github.com/kubernetes-sigs/kubespray) master 브랜치가 업데이트 되면서
python, pip 도 업데이트 한 뒤에 작업 진행해야 오류 안 생기는 거 같아요

저는 아래 기준으로 두니까 잘 됐어요
<img width="376" alt="image" src="https://github.com/user-attachments/assets/f968a041-27de-4e81-b58d-fc842ae00165" />

**Comment by 2Ho0** [comment]
저도 3.10버전으로 업그레이드 했는데
![image](https://github.com/user-attachments/assets/b08540b4-fd76-44c0-a175-76e083a184ce)
여기에서
![image](https://github.com/user-attachments/assets/ae5106cc-bc22-4a9f-a1e7-6f7e816de475)
이런 오류가  나타나서
`ansible-playbook -i inventory/mycluster/hosts.yml --become -e 'ansible_python_interpreter=/usr/bin/python3.8' cluster.yml`
이렇게 ansible은 python3.8로 설정해주니 잘 작동했습니다
